.github/workflows/update-pgxn-version.yml

```
@@ -8,10 +8,11 @@ on:
   workflow_dispatch:
     inputs:
       project_version:
-        description: 'Version number to be released e.g 10.0.3'
+        description: "Version number to be released e.g 10.0.3"
```

```suggestion
        description: "Version number to be released e.g 10.0.3"
        required: true
```

`required` field is not optional here. I suggest you validate your taml files for Github workflows